forecast_weather.py

```diff
@@ -129,21 +153,66 @@ def get_daily_dataframe(response: WeatherApiResponse, station: Station):
         print(f"Timezone {response.Timezone()} {response.TimezoneAbbreviation()}")
         print(f"Timezone difference to GMT+0 {response.UtcOffsetSeconds()} s")
 
-        hourly_df = get_hourly_dataframe(response, station)
+        hourly_df = get_hourly_station_data_frame(response, station)
         hourly_dfs.append(hourly_df)
 
-        daily_df = get_daily_dataframe(response, station)
+        daily_df = get_daily_station_data_frame(response, station)
         daily_dfs.append(daily_df)
 
     hourly_combined = pd.concat(hourly_dfs, ignore_index=True)
     daily_combined = pd.concat(daily_dfs, ignore_index=True)
 
-    hourly_filename = "forecast_weather_hourly_{:%Y-%m-%d}.csv".format(
-        datetime.datetime.now()
+    basename = "forecast_station_weather"
+    hourly_filename = "{}_hourly_{:%Y-%m-%d}.csv".format(
+        basename, datetime.datetime.now()
```

Comment:
Might be good to use now(timezone.utc) WDYT?

Reply:
It's the date and it will be programmed in the cron job anyway.